draft-ietf-mls-protocol.md

```diff
@@ -1511,10 +1511,11 @@ information.  The format of the encoded identity is defined by the application.
 
 For an X.509 credential, each entry in the chain represents a single DER-encoded
 X.509 certificate. The chain is ordered such that the first entry (chain[0]) is
-the end-entity certificate and each subsequent certificate in the chain MUST be
-the issuer of the previous certificate.  The public key encoded in the
+the end-entity certificate and each subsequent certificate in the chain SHOULD be
```

Comment:
I am very uncomfortable with this change for two reasons. Every SHOULD needs to have a statement explaining when you would not do the recommended behavior and why. Also, since this is regarding credential validation I think this is a pretty terrible idea. 

I would prefer to leave this as a MUST. I will comment on another way to accomplish the stated goal in the Conversation part of the PR.